warehouse/azure-synapse/azure-synapse.go

```diff
@@ -611,6 +611,31 @@ func (as *HandleT) loadUserTables() (errorMap map[string]error) {
 	return
 }
 
+//Need to create a structure with delete parameters instead of simply adding a long list of params
+func (as *HandleT) DeleteBy(tableNames []string, jobRunID string, sourceID string, taskRunID string) (success bool, err error) {
+	pkgLogger.Infof("AS: Cleaning up the followng tables in postgres for PG:%s : %v", tableNames)
+	for _, tb := range tableNames {
+		if tb != "rudder_discards" {
```

Comment:
I would refactor it to be following to preserve on identation:

```
if tb == "rudder_discards" {
    continue
}

sql := fmt.Sprintf(...)
```

Reply:
Cool

Reply:
Will be removing this part of code because we are ensuring we are not adding rudder_discards anyway wh_async_jobs